fgpyo/util/logging.py

```diff
@@ -159,10 +159,7 @@ def record_alignment(
         Returns:
             true if a message was logged, false otherwise
         """
-        if rec.reference_start is None:
-            return self.record(None, None)
-        else:
-            return self.record(rec.reference_name, rec.reference_start + 1)
+        return self.record(rec.reference_name, rec.reference_start + 1)
```

Comment:
`AlignedSegment.reference_start` is of type `int`, so the `if` block was unreachable.